active-record/src/test/java/MagicWandSpecification.java

```diff
@@ -0,0 +1,60 @@
+import com.iluwatar.active.record.SpellCastException;
+import com.iluwatar.active.record.WandCoreType;
+import com.iluwatar.active.record.WandWoodType;
+import com.iluwatar.active.record.MagicWand;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by Stephen Lazarionok.
+ */
+public class MagicWandSpecification {
+
+    @Test
+    public void shouldCastFireball_IfMaigPower10AndMore() throws SpellCastException {
+
+        final MagicWand wand = new MagicWand();
+        wand.setWood(WandWoodType.WINE);
+        wand.setCore(WandCoreType.DRAGON_HEARTSTRING);
+        wand.setLengthInches(10.0d);
+
+        Assert.assertTrue(wand.calculateMagicPower() == 10.0d);
```

Comment:
Changed to static import. However why do you recommend using static methods instead of normal ones?